src/Builder.jl

```diff
@@ -136,6 +143,20 @@ function Selectors.runner(::Type{RestoreOutputStreams}, doc::Documents.Document)
     Utilities.restore_output_stream!(doc.internal.stream)
 end
 
+function Selectors.runner(::Type{Populate}, doc::Documents.Document)
+    Utilities.log("populating indices")
```

Comment:
`.` at end of message. Also needs the code block in https://github.com/JuliaDocs/Documenter.jl/blob/master/docs/src/man/guide.md#building-an-empty-document that displays the usual output to be updated to reflect this change:

```
Documenter: setting up build directory.
Documenter: redirecting output streams.
Documenter: expanding markdown templates.
Documenter: building cross-references.
Documenter: running document checks.
Documenter: restoring output streams.
Documenter: rendering document.
Documenter: copying assets to build directory.
```